pkg/handlers/clusterlogging/handler.go

```diff
@@ -1,17 +1,20 @@
 package clusterlogging
 
 import (
+	"fmt"
 	"net/http"
+	"net/url"
+	"strings"
 
 	"github.com/openshift/elasticsearch-proxy/pkg/config"
 	"github.com/openshift/elasticsearch-proxy/pkg/handlers"
 	ac "github.com/openshift/elasticsearch-proxy/pkg/handlers/clusterlogging/accesscontrol"
-	"github.com/openshift/elasticsearch-proxy/pkg/handlers/clusterlogging/types"
 	log "github.com/sirupsen/logrus"
 )
 
 const (
 	clusterLogging = "clusterlogging"
+	kibanaPrefix   = "/.kibana"
```

Comment:
Do you need the preceding slash?

Reply:
no, the path that comes in has it... i can rename the variable to not be as overloaded of a term